### PR TITLE
Contributing.md added

### LINKED ISSUE #8: [📑 Docs]: Add proper README and CONTRIBUTING.md files
Improve the README of this repository by adding proper tags and `Steps to contribute` to this project. Also, add a CONTRIBUTING.md file to list down all the requirements required and the guidelines to follow while contributing to this project.

### PR DESCRIPTION
**Pull request by Priyansh61**
**Description**

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by Priyansh61** [comment]
Hey
I still need to figure out the .env config since i am having some trouble in loading the site locally with current configuration.

I have tried loading database with atlas, generated bearer token for twitter api but still all efforts are in vain and i end up getting a new error each time i use npm start in server side

I dont know what can i do there, so please it would be great if you can share how to set up the environment locally ?

**Comment by Priyansh61** [comment]
hey I have updated the contributing.md , do let me know if any changes needs to be made..

And regarding readme one of my batchmate wanted to make his first contribution, so he has made another PR for readme.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<h1>Contributing to JobiE</h1>
```
Comment (akshatnema) [inline CONTRIBUTING.md]:
```suggestion
<h1>Contributing to JobIE</h1>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<p>:wave: Hope you enjoy contributing to jobie :+1:</p>
```
Comment (akshatnema) [inline CONTRIBUTING.md]:
```suggestion
<p>:wave: Hope you enjoy contributing to JobIE :+1:</p>
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+<h2>Raise an Issue</h2>
+<p>Did you find something which can be improved further ?? Feel free to share it <a href="https://github.com/akshatnema/jobie/issues"> here</a>.</p>
+
+<p>Pull requests are the way concrete changes are made to the code, documentation dependencies, and tools contained in the JobiE repository.</p>
+<h2>Setting up your local environment</h2>
+<h3>Step 1:Fork</h3>
+<p>Fork the project on Github and clone your fork locally to your system.</p>
+
+```console
```
Comment (akshatnema) [inline CONTRIBUTING.md:27]: Don't merge the commands for ssh and HTTPS. Add them separately as:
Using SSH
```bash
  git clone git@github.com:<your_username>/jobie.git
```
OR
Using HTTPS:
```bash
git clone https://github.com/<your_username>/jobie.git
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+```
+<h3>Step 2:Branch</h3>
+To keep your development environment organized, create local branches to hold your work. These should be branched directly off of the main branch.
+
+```console
+   $ git checkout -b my-branch -t upstream/main
+   $ git remote add upstream https://github.com/akshatnema/jobie.git
+   $ git fetch upstream
+```
+<h3>Step 3:Build</h3>
```
Comment (akshatnema) [inline CONTRIBUTING.md:38]: Remove the installation of Nodejs installation from CONTRIBUTING.md file.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+```console
+   $ git add my/changed/file
+   $ git commit -m "changes-made"
+```
+<h3>Step 7:Rebase</h3>
+<p>Once you have committed your changes, it is a good idea to use git rebase (not git merge) to synchronize your work with the main repository.</p>
+
+```console
+   $ git fetch upstream
+   $ git rebase upstream/main
```
Comment (akshatnema) [inline CONTRIBUTING.md]: Remove this rebasing command.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+```
+<p>This ensures that your working branch has the latest changes from main.</p>
+<h2>Testing</h2>
```
Comment (akshatnema) [inline CONTRIBUTING.md]: Presently, there are no tests available to check the linting or bugs in the changes. So, better you right now remove it and we will add this afterwards when tests will be added.

**Review by akshatnema — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (Priyansh61) [reply CONTRIBUTING.md]: Removed the rebasing of branch

**Comment by Priyansh61** [comment]
All the necessary changes have been made, do let me know if anything is left?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+<p><b>ATLAS URI</b>: Click <a href="https://account.mongodb.com/account/login?n=%2Fv2%2F62f7e246576e344fad111a1c&nextHash=%23clusters">here</a>.</p>
+
+<h3>Step 5:Start Developing</h3>
+<p>For all the users working on the frontend. You can make changes and load JobiE locally by running the following inside the web directory.</p>
+
+```console
+   $ npm start
+```
+<p>Similarly for the server side, use the same command inside the sever directory.</p>
+Your site will be hosted locally at: <b>http://localhost:8000/</b>
```
Comment (akshatnema) [inline CONTRIBUTING.md]:
```suggestion
Your site will be hosted locally at: <b>http://localhost:3000/</b>
```

**Review by akshatnema — APPROVED** [review]
LGTM @Priyansh61 🥳

**Comment by Priyansh61** [comment]
Thanks.. Learnt a lot from you, will love to work more with you in future ☺️

**Pull request merged**